app/app/local_settings_old.py

```diff
@@ -0,0 +1,155 @@
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+import os
+
+####################################################################
+# CONFIG
+####################################################################
+
+# SECURITY WARNING: don't run with debug turned on in production!
+#required
+DEBUG = True
+ENV = 'local'
+ALLOWED_HOSTS = [
+    '*'
+]
+BASE_URL = 'http://localhost:8000/'
+
+
+#optional
+TWITTER_USERNAME = 'TODO'
+CONTACT_EMAIL = 'TODO'
+PERSONAL_CONTACT_EMAIL = 'you@foo.bar'
+
+# -------------------------------------------------------
+
+#optional
+TIME_ZONE = 'MST'
+LOGGING = {}
+DEBUG_APPS = []
+STATIC_URL = '/static/'
+
+####################################################################
+# SECRETS
+####################################################################
+
+# SECURITY WARNING: keep the secret key used in production secret!
+# required
+SECRET_KEY = 'TODO'
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'HOST': 'localhost',
+        'PORT': 5432,
+    }
+}
+
+if 'TRAVIS' in os.environ:
+    DATABASES['default']['HOST'] = 'localhost'
+    DATABASES['default']['PORT'] = ''
+
+SENDGRID_API_KEY = 'TODO' # required only needed if you send emails
+SERVER_EMAIL = 'server@TODO.co'
+ADMINS = (
+  ('TODO', 'todo@todo.net'),
+)
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_HOST_USER = 'TODO'
+EMAIL_HOST_PASSWORD = 'TODO'
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+
+# optional:  only if you're syncing web3 (mostly used for bounty flow)
+CUSTOM_MAINNET_GETH_HOST = 'TODO'
+CUSTOM_MAINNET_GETH_PORT = '8545'
+CUSTOM_RINKEBY_GETH_HOST = 'TODO'
+CUSTOM_RINKEBY_GETH_PORT = '8545'
+CUSTOM_TESTRPC_GETH_HOST = 'localhost'
+CUSTOM_TESTRPC_GETH_PORT = '8545'
+TESRPC_CONTRACT_ADDRESS = '0x0ed0c2a859e9e576cdff840c51d29b6f8a405bdd' #run `scripts/testrpc.bash` and `scripts/prepTestRPC.bash` from https://github.com/gitcoinco/smart_contracts
+DEFAULT_NETWORK = 'testrpc'
+INFURA_KEY = 'TODO' # requied only needed if you use infura
+
+CSRF_TRUSTED_ORIGINS = [
+    'localhost'
+]
+
+# optional: only needed if you run the github management commands
+# Login/logout functionality will not work without defining these fields.
+# Setup instructions: https://github.com/gitcoinco/web#setup-github-oauth2-app-integration
+GITHUB_CLIENT_ID = 'b8b56ad694d752c49c1d'
```

Comment:
I'm assuming you didn't mean to commit this.  I'd rekey all of your secrets here. @KennethAshley